TrenDAP.sql

```diff
@@ -95,8 +95,16 @@ CREATE TABLE DataSet
     [Name] VARCHAR(200) NOT NULL,
     [User] VARCHAR(MAX) NOT NULL,
     [Public] bit NULL DEFAULT 0,
-    UpdatedOn DATETIME NULL DEFAULT GETUTCDATE(),
-    [JSON] varbinary(max) NULL
+    UpdatedOn DATETIME NULL DEFAULT GETUTCDATE()
+)
+GO
+
+CREATE TABLE DataSourceDataSet
+(
+    ID INT IDENTITY(1, 1) NOT NULL PRIMARY KEY,
+    DataSourceID INT NOT NULL REFERENCES DataSource(ID),
+    DataSetID INT NOT NULL REFERENCES DataSet(ID),
+    SettingsBin varbinary(max) NOT NULL DEFAULT '0x7B7D'
```

Comment:
This gets translated to a JSON string, but must be stored in varbinary since the max character limit in sql is 4000 characters, which is a bit low depending on the datasource type.